libs/libxslt/stone.yml

```diff
@@ -0,0 +1,38 @@
+#
+# SPDX-FileCopyrightText: © 2020-2023 Serpent OS Developers
+#
+# SPDX-License-Identifier: Zlib
+#
+name        : libxslt
+version     : '1.1.37'
+release     : 1
+homepage    : https://gitlab.gnome.org/GNOME/libxslt/-/wikis/home
+upstreams   :
+    - https://download.gnome.org/sources/libxslt/1.1/libxslt-1.1.37.tar.xz : 3a4b27dc8027ccd6146725950336f1ec520928f320f144eb5fa7990ae6123ab4
+summary     : XSLT processor based on libxml2
+description : |
+    libxslt provides bindings to interact with XSLT files and
+    is based on GNOME's libxml2.
+license     :
+    - X11
+builddeps:
+    - pkgconfig(libxml-2.0)
```

Comment:
It looks like this supports being built against `xz` and `libgcrypt` (which we don't package yet but will sooner than later. Do you happen to know what building with those adds support for?

Reply:
I couldn't find anything about xz in the GitLab repo. For crypto, `libexslt` seems to have some [encryption functions](https://gitlab.gnome.org/GNOME/libxslt/-/blob/master/libexslt/crypto.c).

Reply:
Yeah those look like they are no longer checked for.